src/components/Button.vue

```diff
@@ -33,9 +42,16 @@ export default defineComponent({
       type: Boolean,
       default: false,
     },
+    pauseActiveStyle: {
+      type: Boolean,
+      default: true,
+    },
   },
-  setup: ({}, { emit }) => {
+  setup: (props: Props, { emit }) => {
+    const isActive = ref(false);
+
     const handleClick = (e: any) => {
+      isActive.value = props.pauseActiveStyle ? !isActive.value : false;
```

Comment:
```suggestion
      isActive.value = props.pauseActiveStyle && !isActive.value;
```
booleanは論理演算子を有効活用したほうがわかりやすいのでこっちに